src/openforms/forms/templates/admin/forms/form/export.html

```diff
@@ -0,0 +1,53 @@
+{% extends "admin/base_site.html" %}
+{% load static i18n django_admin_index %}
+
+{% block extrastyle %}{{ block.super }}
+<link rel="stylesheet" href="{% static "admin/css/admin-index.css" %}">
+{% endblock %}
+
+{% block nav-global %}{% include "django_admin_index/includes/app_list.html" %}{% endblock nav-global %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+&rsaquo; <a href="{% url 'admin:forms_form_changelist' %}">{% trans 'Forms' %}</a>
+&rsaquo; {% trans 'Export forms' %}
+</div>
+{% endblock %}
+
+{% block title %} {% trans "Export forms" %} {{ block.super }} {% endblock %}
+
+{% block content %}
+    <h1>{% trans 'Export forms' %}</h1>
+    <div id="content-main">
+        {% blocktrans %}
+            <div>
+                Please enter your email address below. Once your request has been processed, you
+                will receive a link where you can download a zip file with all the exported forms.
+                <br/><br/>
+            </div>
+        {% endblocktrans %}
```

Comment:
Keep HTML out of translatable strings, as this provides an HTML injection risk depending on who manages the translations.

I also think that asking the e-mail address is only needed if there is no `request.user.email`, and would even go a step further to have them configure that in their user account before being able to make exports. That simplifies the amount of edge cases we need to deal with. This could be an early check in the admin action.